plugins/MauticCrmBundle/EventListener/CompanySubscriber.php

```diff
@@ -60,28 +60,42 @@ public static function getSubscribedEvents()
      */
     public function onCompanyPostSave(Events\CompanyEvent $event)
     {
+        $company = $event->getCompany();
+        if ($company->getEventData('pipedrive.webhook')) {
+            // Don't export what was just imported
+            return;
+        }
+
+        /** @var PipedriveIntegration $integrationObject */
         $integrationObject = $this->integrationHelper->getIntegrationObject(PipedriveIntegration::INTEGRATION_NAME);
```

Comment:
@npracht @thecodeine and @Webmecanik - this and all other instances where "Pipedrive" is referred to in a generic CRMBundle file should be removed.